scripts/aem/grub.cfg

```diff
@@ -0,0 +1,37 @@
+set debug=linux,relocator,multiboot,slaunch
+serial --speed=115200 --word=8 --parity=no --stop=1
+terminal_input  serial
+terminal_output serial
+
+insmod part_msdos
+insmod ext2
+set root='hd0,msdos1'
+
+menuentry 'Minimal' {
+    echo    'Loading Linux ...'
+    linux /grub/bzImage placeholder initrd=initrd.img load_ramdisk=1 prompt_ramdisk=0 rw console=vga console=tty0 console=ttyS0,115200 earlyprintk=ttyS0,115200n8
+    echo    'Loading initial ramdisk ...'
+    initrd /grub/initramfs.cpio
+}
+
+menuentry 'Qubes, with Xen hypervisor' {
+    echo    'Loading Xen ...'
+    multiboot2      /grub/xen.gz placeholder console=tty0 console=ttyS0,115200 dom0_mem=min:1024M dom0_mem=max:4096M ucode=scan smt=off gnttab_max_frames=2048 gnttab_max_maptrack_frames=4096 loglvl=all guest_loglvl=all com1=115200,8n1 console=com1 ${xen_rm_opts}
```

Comment:
`xen_rm_opts` are not defined. For slaunch I believe those should be `no-real-mode edd=off`.